### ISSUE TITLE
Use inline styles to reduce complexity

### ISSUE THREAD

**Issue by cgarvis** [vigetlabs/react-ink#3]

**Comment by nhunzaker** [comment]
I actually just bumped into this myself. I think it's a solid idea.

**Comment by cgarvis** [comment]
If you do the styles in a separate file, it almost feels like css

```
// styles.js
module.exports = {
  ink: {
    borderRadius: 'inherit';
    color: 'inherit';
    fill: 'currentColor';
    height: '100%';
    left: '0';
    position: 'absolute';
    top: '0';
    transform: 'translateZ(0)';
    userSelect: 'none';
    width: '100%';
    willChange: 'all';
  }
};
```

Instead of
```
<svg className="ink" style={ this.props.style }>
```

do

```
<svg style={ Object.assign({}, styles.ink, this.props.style}) } >
```

**Comment by nhunzaker** [comment]
@cgarvis I've made an update (`3.0.0`), but before I ship it to `npm` would you be willing to verify? Otherwise, things look all clear to me.

**Comment by cgarvis** [comment]
I just published [react-vendor-prefix](https://github.com/cgarvis/react-vendor-prefix).   This should fix any legacy browser issues you have with `transform`.

**Comment by nhunzaker** [comment]
Ah very cool! Actually I think we can get away with not having transform. After some testing, there doesn't seem to be a benefit and I think I'd rather let the developer using `<Ink />` make the judgement call on whether or not they want the SVG element to render on a separate composite layer.

I've also published this at version `3.0.0`
